### PR TITLE
add clang format and git

### PR DESCRIPTION
**Pull request by fusuiyi123**

### CODE REVIEW

File: travis-checks/Dockerfile [hunk 1]
```diff
@@ -14,7 +14,9 @@ RUN apt-get install -y \
     autoconf \
     automake \
     make \
-    cmake
+    cmake \
+    clang-format \
+    git
```
Comment (gavv) [inline travis-checks/Dockerfile]: You've added them to `for 3rd-parties` section, but that's not quite correct. Let's introduce a new section, say `for scons fmt`.

**Comment by fusuiyi123** [comment]
added another sec

**Comment by gavv** [comment]
Thanks!

**Pull request merged**